### PR TITLE
Init Layout

### PR DESCRIPTION
**Pull request by anhpt-0250**
## Related Tickets
https://edu-redmine.sun-asterisk.vn/issues/67395

## WHAT (optional)
- Layout cho user + admin

## HOW
- Theo tutorial + PR của chị

## Evidence (Screenshot or Video)
Screenshot from 2023-09-05 16-06-40
Screenshot from 2023-09-05 16-07-01
Screenshot from 2023-09-05 16-07-23

### CODE REVIEW

**Comment by leebaoanh** [comment]
ok

**Comment by anhpt-0250** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* mixins, variables, etc. */
+
+$gray-medium-light: #eaeaea;
```
Comment (ngocvt-0484) [inline app/assets/stylesheets/application.scss:6]: tạo file lưu mã màu

File: app/views/layouts/_header.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<header class="navbar navbar-fixed-top navbar-inverse">
+  <div class="container">
+    <%= link_to "Logo", root_path, id: "logo" %>
+    <nav>
+      <ul class="nav navbar-nav navbar-right">
+        <li><%= link_to "Home", root_path %></li>
```
Comment (ngocvt-0484) [inline app/views/layouts/_header.html.erb:6]: i18n đâu hết rồi

File: app/views/layouts/application_admin.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'application_admin', media: 'all', 'data-turbolinks-track': 'reload' %>
+  </head>
+
+  <body>
+      <%= render "layouts/header_admin" %>
+      <%= yield %>
+
```
Comment (ngocvt-0484) [inline app/views/layouts/application_admin.html.erb:17]: xóa dòng thừa

File: app/views/layouts/application_admin.html.erb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+<!-- dung application_admin.js, application_admin.css -->
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%= full_title(yield(:title)) %></title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+    <%= stylesheet_link_tag 'application_admin', media: 'all', 'data-turbolinks-track': 'reload' %>
```
Comment (ngocvt-0484) [inline app/views/layouts/application_admin.html.erb:11]: nháy "

**Comment by anhpt-0250** [comment]
ready

**Pull request merged**